internal/daemon/daemon.go

```diff
@@ -328,6 +328,21 @@ func (d *Daemon) initServer(resources ...rest.Resources) *http.Server {
 	mux.SkipClean(true)
 	mux.UseEncodedPath()
 
+	state := d.State()
```

Comment:
this changes the order of route definitions so that if a handler was defined for / in a downstream app, that handler would be used instead.

context:
In site manager, if / is accessed from a browser client, we want to redirect that request to /ui. This is also implemented in LXD.